app/src/main/java/com/appculinaryrecipes/fragments/AddRecipeFragment.java

```diff
@@ -256,15 +247,14 @@ public void onSuccess(UploadTask.TaskSnapshot taskSnapshot) {
                         fragment).commit();
     }
 
-    public void fetchApiKey(String collectionName, String documentName){
+    public void fetchApiKey(String collectionName, String documentName) {
         FirebaseFirestore database = FirebaseFirestore.getInstance();
         DocumentReference documentReference = database.collection(collectionName).document(documentName);
         documentReference.get().addOnCompleteListener(task -> {
```

Comment:
tutaj zamiast addOnCompleteListener można używać addSnapshotListener, wtedy będzie pobierało dane jak zostanie wprowadzona jakaś zmiana w bazie. Coś jak lifecycle

Reply:
A to działa na Firestore czy na Real Time ?

Reply:
Na Firestore normalnie

Reply:
https://firebase.google.com/docs/firestore/query-data/listen